torch/distributed/_sharded_tensor/ops/linear.py

```diff
@@ -130,16 +142,20 @@ def _handle_col_wise_sharding(input, world_size, weight, local_shard_t, bias, pg
 
     Returns: final result of linear operation.
     """
-    return _handle_col_wise_sharding_base(
-        torch.matmul,
-        weight.size(0),
-        len(input.size()) - 1,
-        input,
-        world_size,
-        weight,
-        local_shard_t,
-        pg,
-    ) + bias
+    return (
+        _handle_col_wise_sharding_base(
+            torch.matmul,
+            weight.size(0),
+            len(input.size()) - 1,
+            input,
+            world_size,
+            weight,
+            local_shard_t,
+            pg,
+        )
+        + bias
+    )
```

Comment:
Usually its better to keep such formatting changes in a separate PR so it is easier to review.

Reply:
Sure. It's because the auto-format is applied to the whole doc while originally I just want to format the new code I added. I think there is a script which we can use to format all code under distributed folder so that we will not run into this kind of situation that often.